### DISCUSSION TITLE
Lost all content after update

### DISCUSSION THREAD

**Discussion by NightHawk1010** [GetPublii/Publii#1461] [Help] [answered]
Hi,
I have a live website (johnwright.no) created with Publii. After an update everything is gone when I start the desktop app on my Mac. I updated to my new MAC with M3. Is there a way to get the live web back into the app? I am greatful for any advise.

**Comment by 3muchfun** [comment]
Do you have access to your old machine? If so, I would suggest to make a back up within Publii and import it to your new Publii installation.
Never forget the golden rule: Save soon, safe often.

**Reply by NightHawk1010** [reply]
Thanks, old machine is empty. :)

**Comment by candidexmedia** [comment]
All your computer files are gone?? 😳

**Reply by NightHawk1010** [reply]
No, files seems to be in another folder. I don't know how to fix that.

**Reply by NightHawk1010** [reply, marked as answer]
Well... it was easy. I just copied the hole site into the new folder. I should have discovered that before asking here. Sorry.

**Reply by candidexmedia** [reply]
Fiouf, thank goodness!

For your reference: https://getpublii.com/docs/publii-global-settings.html#fileslocation

Remember to always make backups (preferably to a cloud storage).
